### PR TITLE
- add parallel option to build_sln_command

### PR DESCRIPTION
**Pull request by SSE4**
follow up #1390

### CODE REVIEW

File: conans/client/tools/win.py [hunk 1]
```diff
@@ -5,13 +5,14 @@
...
 from conans.client.tools.files import unix_path
+from conans.client.tools.oss import cpu_count
 from conans.errors import ConanException
 
 _global_output = None
 
 
 def msvc_build_command(settings, sln_path, targets=None, upgrade_project=True, build_type=None,
-                       arch=None):
+                       arch=None, parallel=True):
```
Comment (lasote) [inline conans/client/tools/win.py]: Not used

**Comment by lasote** [comment]
Missing tests

**Comment by SSE4** [comment]
@lasote  k, will add once boot to Windows

File: conans/client/tools/win.py (hunk 1 above)
Comment (SSE4) [reply conans/client/tools/win.py]: fixed

**Comment by SSE4** [comment]
@lasote I've added some tests, lemme know if something else has to be tests

File: conans/test/util/build_sln_command_test.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+# -*- coding: utf-8 -*-
+
+import unittest
+from conans.tools import build_sln_command, cpu_count
+from conans.errors import ConanException
+from conans.model.settings import Settings
+from nose.plugins.attrib import attr
+
+
+@attr('visual_studio')
```
Comment (memsharded) [inline conans/test/util/build_sln_command_test.py]: Actually, this test doesn't require VS at all. The tests are green in Linux, even if there is no check to discard "visual_studio" tests.

As the command is not being called, this test work in Linux too. I think the attr can be removed.

**Comment by memsharded** [comment]
Very nice and clean tests, thanks!

File: conans/test/util/build_sln_command_test.py (hunk 2 above)
Comment (SSE4) [reply conans/test/util/build_sln_command_test.py]: my idea was to group/categorize tests, so Visual Studio integration can be checked without running all other tests. I'll remove if it's not needed.

**Pull request merged**